docs/how-to/configure.md

```diff
@@ -104,6 +104,7 @@ Grafana’s [anonymous usage statistics](https://grafana.com/docs/grafana/latest
 As well as via the *PMM Settings* page, you can also disable telemetry with the `-e DISABLE_TELEMETRY=1` option in your docker run statement for the PMM Server.
 
 !!! note alert alert-primary ""
+    - If the Security Threat Tool is enabled in PMM Settings, Telemetry is automatically enabled.
```

Comment:
https://github.com/percona/pmm-doc/commit/4f91785dda1788fcdbb85ae345dfa5a869c6b31d
so, I don't understand do we need it or not ? @fabio-silva

Reply:
that was landed explicitly to 2.25.0

Reply:
indeed, this no longer applies @catalinaadam, we can remove it